benchmarks/run_mo_kurobako.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import os
+import subprocess
+
+
+def run(args: argparse.Namespace) -> None:
+    kurobako_cmd = os.path.join(args.path_to_kurobako, "kurobako")
+    subprocess.run(f"{kurobako_cmd} --version", shell=True)
+
+    if not (os.path.exists(args.data_dir) and os.path.isdir(args.data_dir)):
+        raise ValueError(f"Data directory {args.data_dir} cannot be found.")
+
+    os.makedirs(args.out_dir, exist_ok=True)
+    study_json_filename = os.path.join(args.out_dir, "studies.json")
+    subprocess.check_call(f"echo >| {study_json_filename}", shell=True)
+    solvers_filename = os.path.join(args.out_dir, "solvers.json")
+    subprocess.check_call(f"echo >| {solvers_filename}", shell=True)
+    problems_filename = os.path.join(args.out_dir, "problems.json")
+    subprocess.check_call(f"echo >| {problems_filename}", shell=True)
```

Comment:
Can be simplified as #3468 do so.
```suggestion
    solvers_filename = os.path.join(args.out_dir, "solvers.json")
    problems_filename = os.path.join(args.out_dir, "problems.json")
    
    # Ensure all files are empty.
    for filename in [study_json_filename, solvers_filename, problems_filename]:
        with open(filename, "w"):
            pass
```